cmd/gobl.ksef/send.go

```diff
@@ -146,3 +174,24 @@ func sleepContext(ctx context.Context, delay time.Duration) {
 	case <-time.After(delay):
 	}
 }
+
+func saveFile(name string, data []byte) error {
+	file, err := os.Create(name)
+	if err != nil {
+		return err
+	}
+	defer func() {
+		if err := file.Close(); err != nil {
+			fmt.Println("Error when closing:", err)
+		}
+	}()
+	_, err = file.Write(data)
+	return err
+}
+
+func filename(inv *bill.Invoice) string {
+	if inv.Series != "" {
+		return inv.Series + "-" + inv.Code + ".xml"
+	}
+	return inv.Code + ".xml"
```

Comment:
Why not have one function that does the same thing?
Are we consistent? Look at [SequentialNumber](https://github.com/invopop/gobl.ksef/blob/main/invoice.go#L85)

Reply:
It is a part of a different package and the method you are mentioning isn't yet merged in a different PR.